scripts/tasks/webpack-resources.js

```diff
@@ -52,13 +53,20 @@ module.exports = {
 
     if (isProduction) {
       configs.push(merge({
+        mode: 'production',
         output: {
           filename: `[name].min.js`,
           path: path.resolve(process.cwd(), 'dist')
         },
+
+        optimization: {
+          concatenateModules: true,
+          minimize: true
+        },
```

Comment:
these are the defaults for mode:production, you can omit this

Reply:
If you aren't seeing dev mode work, you probably need to update merge to make sure that you are deep copying all properties in the config when merging. We had a similar issue where our "merge" ended up with a reference to the same optimization object for development and production mode, which caused the defaults webpack applies in the devmode run to get applied in production mode.

Reply:
Good catch, I know about `mode` but i was testing these individually,.